code/src/reducers/labyrinth.js

```diff
@@ -0,0 +1,69 @@
+import { createSlice } from '@reduxjs/toolkit';
+import ui from './ui';
+
+const initialState = {
+  username: null,
+  position: null,
+  actions: '',
+  history: []
```

Comment:
This doesn't look like it's being used, so maybe delete if you don't plan on adding it.